js/controller.js

```diff
@@ -1472,6 +1474,32 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.SAVE_PLAYER_SETTINGS, this.state.persistentSettings);
     },
 
+    /**
+     * Used to enable or disable the CSS workaround that prevents anamorphic videos from
+     * being distorted on Firefox. The fix will only be enabled if ots_stretch_to_output
+     * is set to true in the player attributes.
+     * Note that currently the oo-anamorphic class has effect only on Firefox.
+     * @param {boolen} enabled A value that determines whether to enable or disable the anamorphic videos CSS fix.
+     */
+    trySetAnamorphicFixState: function(enabled) {
```

Comment:
I think this `trySetAnamorphicFixState` would be easier understood as two separate functions because the function does not necessarily do what the function name suggests depending on the `enabled` argument. Perhaps we should split it into `enableAnamorphicFixState` and `disableAnamorphicFixState`? That way, it is also immediately clear to the reader what the function is doing as opposed to some ambiguity of `trySetAnamorphicFixState(true)` and `trySetAnamorphicFixState(false)`